tests/fast-integration/skr-test/test.js

```diff
@@ -31,7 +34,7 @@ describe('SKR test', function() {
   });
 
   // Run the OIDC tests
-  oidcE2ETest(options, getShootInfoFunc);
+  oidcE2ETest(getShootOptionsFunc, getShootInfoFunc);
```

Comment:
For reviewer:
I updated it to `getShootOptionsFunc` because the `options` was evaluated and passed to the method before the `before` section was executed.